lib/magma/server/update.rb

```diff
@@ -83,16 +83,15 @@ def update_any_file_links
     @revisions.each do |model, model_revisions|
       model_revisions.each do |revision|
 
-        revision.attribute_names.each do |attribute|
-          if is_file_attribute(revision.model, attribute)
-
-            if !revision.to_loader[attribute]  # nil
-              remove_copy_on_metis(revision, attribute)
-            elsif revision.to_loader[attribute][:location].start_with? 'metis:'
-              copy_file_on_metis(revision, attribute)
-            elsif revision.to_loader[attribute][:location] == '::blank'
-              remove_copy_on_metis(revision, attribute)
-            end
+        revision.attribute_names.select {
+          |attribute| is_file_attribute(revision.model, attribute)
+        }.each do |attribute|
```

Comment:
seems trivial but I would use attribute_name here, it quickly becomes confusing whether you are dealing with the name or the actual object.